UnityProject/Assets/Scripts/Core/Lighting/LightSource.cs

```diff
@@ -261,8 +261,8 @@ private void TryRemoveBulb(HandApply interaction)
 	{
 		var handSlot = interaction.PerformerPlayerScript.ItemStorage.GetNamedItemSlot(NamedSlot.hands);
 
-		if (mState == LightMountState.On && handSlot.IsOccupied &&
-				!Validations.HasItemTrait(handSlot.ItemObject, CommonTraits.Instance.BlackGloves))
+		if (mState == LightMountState.On && (!handSlot.IsOccupied ||
```

Comment:
I hate it but use `` == false `` instead. It is the style we should be enforcing.